ui/app/templates/components/database-role-edit.hbs

```diff
@@ -95,7 +95,7 @@
         @roleType={{@model.type}}
         @model={{@model}}
         @mode={{@mode}}
-        @dbType={{this.databaseType}}
+        @dbType={{await this.databaseType}}
```

Comment:
I'm all about adding this helper with ember-promise-helpers. I think you were there for the ui-guild meeting where we discussed the argument against having promises in templates but I think we have enough use cases for it. Happy to see it implemented!